paket.dependencies

```diff
@@ -15,4 +15,4 @@ nuget FSharp.Core 4.7.0 #
 
 
 group git
-    git file:///C:/users/thomas/git/cwtools master
\ No newline at end of file
+    git https://github.com/cwtools/cwtools.git master
```

Comment:
I have this in order to make local development possible. It's not a practical workflow to push to github to test new features :P.
If you have any ideas on how to have a local override for this, that'd be amazing. I suspect it needs development on paket itself to support such a feature though!

Reply:
--group, -g <name>    restore dependencies of a single group
There can be 2 groups, one for local, and for non-local restore?